shell/browser/osr/osr_web_contents_view.h

```diff
@@ -104,7 +105,7 @@ class OffScreenWebContentsView : public content::WebContentsView,
   raw_ptr<content::WebContents> web_contents_ = nullptr;
 
 #if BUILDFLAG(IS_MAC)
-  OffScreenView* offScreenView_;
+  RAW_PTR_EXCLUSION OffScreenView* offScreenView_;
```

Comment:
Why was this excluded ? A tracking issue with the error would be good.

Reply:
Converting it throws a compile error.  I'll check what that error is.

Reply:
@deepak1556 the error is:
```
../../base/allocator/partition_allocator/pointers/raw_ptr.h:562:17: error: static assertion failed due to requirement 'raw_ptr_traits::IsSupportedType<OffScreenView, void>::value': raw_ptr<T> doesn't work with this kind of pointee type T
  static_assert(raw_ptr_traits::IsSupportedType<T>::value,
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../electron/shell/browser/osr/osr_web_contents_view.h:108:26: note: in instantiation of template class 'base::raw_ptr<OffScreenView>' requested here
  raw_ptr<OffScreenView> offScreenView_;
```

It looks like this is the same case as ElectronMenuController below.  It is a pointer to a Objective-C object:
In https://source.chromium.org/chromium/chromium/src/+/main:base/memory/raw_ptr.md under the section titled `When it is okay to continue using raw C++ pointers -> Unsupported cases leading to compile errors`, it lists `Pointers to Objective-C objects`, which is what OffScreenView is, so it can't be converted and thus does not need a tracking issue.